.github/workflows/ci-chaincli.yml

```diff
@@ -0,0 +1,26 @@
+name: chaincli CI
+
+on:
+  push:
+    paths:
+      - "core/scripts/chaincli/**"
+  pull_request:
+    paths:
+      - "core/scripts/chaincli/**"
+
+jobs:
+  golangci:
+    if: ${{ github.event_name == 'pull_request' ||  github.event_name == 'schedule' }}
+    name: chaincli-lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Golang Lint
+        uses: ./.github/actions/golangci-lint
+        with:
+          name: chaincli-lint
+          go-directory: core/scripts/chaincli
```

Comment:
Is it sane to only set this and have the other 2 args proceeding be inferred?

Reply:
Do you mean remove the `go-version-file` and `go-module-file` and refer with just `go-directory`?

Reply:
yeah exactly :)

Reply:
Ahh if we go that way I think we still need to do some hacky stuff to remove the `chaincli` part of the `go-directory` since the `go.sum` and `go.sum` is one level up.

The main reason for the `go-directory` is because of this:
```
    - name: Build binary
      if: ${{ inputs.go-directory != '.' }}
      working-directory: ${{ inputs.go-directory }}
      shell: bash
      run: go build
      ```
to build the image.